composer.json

```diff
@@ -66,10 +66,6 @@
                 "https://www.drupal.org/files/issues/panelizer-panels-ipe-tempstore-id.patch"
             },
             "drupal/panels": {
-                "2747739 - Mega Patch":
-                "https://www.drupal.org/files/issues/panels-beta4-75131285.patch",
```

Comment:
This mega patch needs update. We need to find out what's not yet merged in beta5 and add the rerolled patch